lib/app.js

```diff
@@ -27,5 +27,10 @@
     };
   }
 
+  ko.components.register('calculator', {
```

Comment:
Register the 'calculator' component, telling knockout where to find the JS and HTML for the component